torch/profiler/_memory_profiler.py

```diff
@@ -1176,8 +1176,8 @@ def export_memory_timeline_html(
         title = "\n\n".join(
             ([title] if title else [])
             + [
-                f"Max memory allocated: {max_memory_allocated/(10**9):.2f} GB \n"
-                f"Max memory reserved: {max_memory_reserved/(10**9):.2f} GB"
+                f"Max memory allocated: {max_memory_allocated/(1024**3):.2f} GB \n"
```

Comment:
Nit: we may want to switch to "GiB" for clarity here: https://en.wikipedia.org/wiki/Byte#Multiple-byte_units

Reply:
Thanks @eqy. I will update the PR. On a side note, I see GB and GiB are being used interchangeably though they are technically different. For instance, NVIDIA uses GB (A100 80GB (https://www.nvidia.com/en-in/data-center/a100/)) though they meant 80 GiB which is around 86GB. I will update the PR with GiB, I am sure that will be more clear. Thank you.